css/house-style.md

```diff
@@ -691,33 +688,33 @@ a { color: pink; }
 A quick note on single line CSS. We allow single line CSS when there is only one declaration, but we shouldn't always use it. We only ever use single line CSS when there are groups of multiple CSS classes with one declaration, for example spacing, to aid legibility. In this case we also do not require a space between classes. So we would use it in the following example:
 
 ```css
-.ma0 { margin: 0; }
-.ma1 { margin: 1px; }
-.ma2 { margin: 2px; }
-.ma4 { margin: 4px; }
-.ma6 { margin: 6px; }
-.ma10 { margin: 10px; }
-.ma15 { margin: 15px; }
-.ma20 { margin: 20px; }
-.ma30 { margin: 30px; }
-.ma40 { margin: 40px; }
-.ma50 { margin: 50px; }
-.ma70 { margin: 70px; }
+.c-ma0 { margin: 0; }
```

Comment:
Out of curiosity, does this go against the previously described rules around naming?
E.g. we say we use `c-button` not `c-btn`.  But here the example is `c-ma0`, rather than `c-margin0`